### PR TITLE
Erica draft

### PR DESCRIPTION
**Pull request by ebirving**

### CODE REVIEW

File: presentations.md [hunk 1]
```diff
@@ -1,34 +1,36 @@
 # Presentations
 
-In place of doing science-fair-style or one-by-one presentations, you will prepare a short (3 -5 minute) presentation which you will record and upload to YouTube. We'll watch the presenations on Friday morning - with popcorn!
+In place of doing science-fair-style or one-by-one presentations, you will prepare a short (3 -5 minute) screencast demo of your project, which you will upload to YouTube/Vimeo. We'll watch the presentations on Friday afternoon - with popcorn!
```
Comment (andrewsunglaekim) [inline presentations.md:4]: make popcorn a suprise!

**Comment by andrewsunglaekim** [comment]
:+1:  looks great erica, just needs schedule for mini lessons

File: schedule.md [hunk 2]
```diff
@@ -1,45 +1,67 @@
...
+
+**Attendance will be taken at Presentations (1:00pm).**
+
+| Time | Event |
+|------|-------|
+| 11:00am | **[Presentation Video](./presentations.md) due** |
+| 12:30-1:00pm | Catered Lunch |
+| 1:00-4:00pm  | [Project Presentations](presentations.md)|
+| 4:00-4:30pm | Feedback Survey |
+| 4:30-5:00pm | Squad Retro! |
```
Comment (andrewsunglaekim) [inline schedule.md]: needs to contain a blurb about overarching one-on-one schedule once we establish it.

File: wdi6_presentation_videos.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+- [Laura N](http://youtu.be/6xT5tiIZNJ4)
+- [Ksenia C](https://youtu.be/VXzvubrKStk)
+- [Omari M](https://youtu.be/7V1qN4hg9Pk)
+- [Arash S](https://www.youtube.com/watch?v=t9YsRogO8S0)
+- [Melissa S](https://www.youtube.com/watch?v=p9XcmIQslpQ&feature=youtu.be)
+- [Lisa S](http://youtu.be/m3yJKUI9nvk)
+- [Adrian S](http://youtu.be/X0Q0LsmiGag)
+- [Leigh V](https://youtu.be/Ov3rY2EtU5w)
+- [Rachel W](https://youtu.be/KV-lujD3gq8)
+- [Yvonne W](https://youtu.be/Nm89-IpSxRk)
```
Comment (andrewsunglaekim) [inline wdi6_presentation_videos.md:50]: this is cool, we should keep this for students fo sho

File: amas.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+Take a break from your project, and ask the instructors anything that's been on your mind, from "Who's your favorite Power Ranger?" to "What's the biggest challenge you've faced as a programmer?" to "What's the meaning of life?" Seriously...ask us anything!
+
+## Schedule
+
+| When | Who | Where |
+|--------|---------------------|
+| Monday, 1:30-2:30pm | Nick, Adam & Adrian | Classroom 5 |
+| Wednesday, 1:30-2:30pm | John, Jesse & Andy | " |
+| Friday, 1:30-2:30pm | Erica, Matt & Robin | " |
```
Comment (adambray) [inline amas.md]: Starting to wonder if we should do 2 sessions, with 4 & 5 instructors respectively? Just so students only have to go to 2 to hear from all of us.

File: asking_for_help.md [hunk 5]
```diff
@@ -1,18 +1,17 @@
 # How to Ask Questions
 
-https://wdidc7.hackpad.com/Instructor-Scrum-1029-5pm-lKTSL4z7NV2
-
 ## TLDR (Too Long; Didn't Read)
-The instructors won't be answering questions in-person this week. Instead, we'll be responding to Github issues on the `pbj-project2` repository during business hours.
+The instructors will be using Github issues on the `project2` repository as the primary forum for answering technical questions. 90% of the time, at least one instructor will be dedicated to monitoring issues at all times during business hours. The other 10% will be announced in advance on Slack.
```
Comment (adambray) [inline asking_for_help.md:4]: :+1:

File: asking_for_help.md [hunk 6]
```diff
@@ -1,18 +1,17 @@
...
 Issues should follow this format:
 ```
 I tried ___.
 I expected ___ to happen.
 ___ happened instead.
+My repo link is ___ and my question is about lines ___.
 ```
 
-[Just click this handy link to create an issue with these blanks inserted for you!](https://github.com/ga-dc/pbj-project2/issues/new?title=issue+with+____&body=I+tried+_____.+I+expected+_____+to+happen._____+happened+instead.)
+[Just click this handy link to create an issue with these blanks inserted for you!](https://github.com/ga-dc/project2/issues/new?title=issue+with+____&body=I+tried+_____.+I+expected+_____+to+happen._____+happened+instead.%0DMy+repo+link+is+___+and+my+question+is+about+lines+___.)
```
Comment (adambray) [inline asking_for_help.md:14]: Nice!

File: one-on-ones.md [hunk 7]
```diff
@@ -0,0 +1,25 @@
+# Project 2 One-on-One Objectives
+
+## Friday (15 minutes)
+- Communicate and lingering concerns and/or confusion about Unit 2 topics.
+- Identify at least one topic to review or research over the weekend in preparation for Project 2
+- Discuss possible project ideas and identify strong candidates
```
Comment (adambray) [inline one-on-ones.md]: I like all three of these, but it's a lot for 15 minutes, I suspect we should maybe just do the last 2 instead? Save Unit 2 questions for review sessions?

File: one-on-ones.md [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+
+### Homework:
+
+- Create a new repository on GitHub for your project.
+  - Create `readme.md` file in the root of your repo with a a 1-2 sentence summary of your project idea.
+  - Create a `planning` directory in the root of your repo, containing...
+    - A `wireframes` directory, inside of which are *at least* 2 photos or drawings of wireframes
+    - A `user_stories.md` file, containing at least 7 user stories, *or* a link to Trello with at least 7 user stories.
```
Comment (adambray) [inline one-on-ones.md]: We should add a photo of an ERD in here as well.

File: one-on-ones.md [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+- Create an issue on *this* repository with a link the repository you created above by **9:00am on Monday, November 16<sup>th</sup>**.
+
```
Comment (adambray) [inline one-on-ones.md:4]: We should probably cover how to do `rails new my_app .` to create a rails app in the current folder, rather than nested.

File: one-on-ones.md [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+## Monday (15 minutes)
+- Review planning documents for feasibility and completeness
+- Delineate the scope of "MVP" for the student's project
+- Set goals for Monday and Tuesday
+- Identify first steps toward achieving Monday's goals
+- Identify potential pitfalls and challenges for the week ahead
```
Comment (adambray) [inline one-on-ones.md:10]: :+1:

File: presentations.md [hunk 11]
```diff
@@ -1,34 +1,36 @@
...
-- A website or e-mail address through which you can be contacted. This can be included either in the video itself or as the video description (the text beneath YouTube videos).
-- A description and demo of your app
-- A code sample explaining one thing you're proud of
-- One thing you'd do differently next time
-- The next feature you would like to implement
+- ...a website or e-mail address through which you can be contacted. This can be included either in the video itself or as the video description (the text beneath YouTube videos).
+- ...a description and demo of your app
+- ...a code sample explaining one thing you're proud of
+- ...one thing you would do differently next time
+- ...the next feature you would like to implement
```
Comment (adambray) [inline presentations.md:16]: - Your voice loud enough to hear! Speak up!

File: presentations.md [hunk 12]
```diff
@@ -1,34 +1,36 @@
...
 
-### Submission deadline: Thursday, August 6th, 11:00pm
+### Submission deadline: Friday, November 20<sup>th</sup>, 11:00am
 
 "But instructors, I won't be done with my project by then!"
 
-That's OK! The people viewing your video don't have to know that. Show off what does work!
+a) That's OK, the people viewing your video don't have to know that. Show off what does work!
+b) Go to sleep!
```
Comment (adambray) [inline presentations.md]: Does this make sense if the deadline is 11am now?

File: readme.md [hunk 13]
```diff
@@ -2,39 +2,42 @@
...
-### Necessary Deliverables
+### Deliverables
 
-* A **working full-stack application, built by you**, hosted somewhere on the internet
-* A **link to your hosted working app** in the URL section of your Github repo
-* A **git repository hosted on Github**, with a link to your hosted project,  and frequent commits dating back to the **very beginning** of the project. Commit early, commit often.
+* A **working full-stack application, built by you**, that meets the technical requirements above.
+* A **git repository hosted on Github**, with **frequent commits dating back to the very beginning** of the project.
+* A **link to your hosted, working application** in the URL section of your Github repo.
+![The URL section of your Github repo](http://i.imgur.com/QQ7RsfR.gif)
```
Comment (adambray) [inline readme.md:38]: nice addition!

File: readme.md [hunk 14]
```diff
@@ -2,39 +2,42 @@
...
 * **A ``readme.md`` file** in the root of your repo, with explanations of the technologies used, the approach taken, installation instructions, unsolved problems, etc.
-* **A `planning` folder** in the root of your repo, inside of which is...
+* **A `planning` directory** in the root of your repo, containing...
   * A `wireframes` directory, inside of which are *at least* 2 photos or drawings of wireframes
   * A `user_stories.md` file, containing at least 7 user stories, *or* a link to Trello with at least 7 user stories.
```
Comment (adambray) [inline readme.md:43]: Plus a photo of your ERD

File: amas.md (hunk 4 above)
Comment (ebirving) [inline amas.md]: Yeah! Just Monday and Wednesday, maybe? Add Robin to Monday, me and Matt to Wednesday?

**Comment by adambray** [comment]
:+1:
